src/main/java/frc/robot/maps/RobotMap.java

```diff
@@ -32,10 +32,15 @@ public IntakeMap getIntakeMap() {
         public SendableSpeedController shooterWheel() {
             return new MockSpeedController();
         }
+
+        public double shooterHeight() {
+            return 45;
```

Comment:
This should default to 0 or not have a default at all. This value should be in "Tempest Map"